docs/en/sql-reference/sql-functions/table-functions/files.md

```diff
@@ -205,21 +190,26 @@ If StarRocks fails to unionize all the columns, it generates a schema error repo
 
 -->
 
-### unload_data
+### unload_data_param
 
 From v3.2 onwards, FILES() supports defining writable files in remote storage for data unloading. For detailed instructions, see [Unload data using INSERT INTO FILES](../../../unloading/unload_using_insert_into_files.md).
 
-- `compression` (Required): The compression method to use when unloading data. Valid values:
-  - `uncompressed`: No compression algorithm is used.
-  - `gzip`: Use the gzip compression algorithm.
-  - `brotli`: Use the Brotli compression algorithm.
-  - `zstd`: Use the Zstd compression algorithm.
-  - `lz4`: Use the LZ4 compression algorithm.
-- `max_file_size`: The maximum size of each data file when data is unloaded into multiple files. Default value: `1GB`. Unit: B, KB, MB, GB, TB, and PB.
-- `partition_by`: The list of columns that are used to partition data files into different storage paths. FILES() extracts the key/value information of the specified columns and stores the data files under the storage paths featured with the extracted key/value pair. For further instructions, see Example 5.
-- `single`: Whether to unload the data into a single file.  Valid values:
-  - `true`: The data is stored in a single data file.
-  - `false`: The data is stored in multiple files if `max_file_size` is reached.
+```sql
+-- Supported from v3.2 onwards.
+unload_data_param::=
+    "compression" = "<compression_method>",
+    "max_file_size" = "<file_size>",
+    "partition_by" = "<column_name> [, ...]" 
+    "single" = { "true" | "false" } 
+```
+
+
+| **Key**          | **Required** | **Description**                                              |
+| ---------------- | ------------ | ------------------------------------------------------------ |
+| compression      | Yes          | The compression method to use when unloading data. Valid values:<ul><li>`uncompressed`: No compression algorithm is used.</li><li>`gzip`: Use the gzip compression algorithm.</li><li>`brotli`: Use the Brotli compression algorithm.</li><li>`zstd`: Use the Zstd compression algorithm.</li><li>`lz4`: Use the LZ4 compression algorithm.</li></ul>                  |
+| max_file_size    | No           | The maximum size of each data file when data is unloaded into multiple files. Default value: `1GB`. Unit: B, KB, MB, GB, TB, and PB. |
+| partition_by     | No           | The list of columns that are used to partition data files into different storage paths. Multiple columns are separated by commas (,). FILES() extracts the key/value information of the specified columns and stores the data files under the storage paths featured with the extracted key/value pair. For further instructions, see Example 5. |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/3742800d-9a81-49d3-b629-192b1e58f6b2)
It's better to give a link here.

Reply:
There are no headings for examples.